packages/@aws-cdk/aws-eks/lib/kubectl-handler/helm/__init__.py

```diff
@@ -51,6 +65,18 @@ def helm_handler(event, context):
             f.write(json.dumps(values, indent=2))
 
     if request_type == 'Create' or request_type == 'Update':
+        # Ensure chart or chart_asset_url are set
+        assert(chart != None or chart_asset_url != None)
```

Comment:
An error message here would help diagnose problems in production.

Reply:
Done